app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -370,56 +369,56 @@ class DefaultToolbarMenu(
     private val newCoreMenuItems by lazy {
         val newTabItem = BrowserMenuImageText(
             context.getString(R.string.library_new_tab),
-            R.drawable.ic_bookmark_filled,
-            disabledTextColor()
+            R.drawable.ic_new,
+            primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.NewTab)
         }
 
         val bookmarksItem = BrowserMenuImageText(
             context.getString(R.string.library_bookmarks),
             R.drawable.ic_bookmark_filled,
-            disabledTextColor()
+            primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.Bookmarks)
         }
 
         val historyItem = BrowserMenuImageText(
             context.getString(R.string.library_history),
             R.drawable.ic_history,
-            disabledTextColor()
+            primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.History)
         }
 
         val downloadsItem = BrowserMenuImageText(
             context.getString(R.string.library_downloads),
             R.drawable.ic_download,
-            disabledTextColor()
+            primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.Downloads)
         }
 
         val extensionsItem = BrowserMenuImageText(
             context.getString(R.string.browser_menu_extensions),
             R.drawable.ic_addons_extensions,
-            disabledTextColor()
+            primaryTextColor()
         ) {
             onItemTapped.invoke(ToolbarMenu.Item.AddonsManager)
         }
 
-        val syncedTabsItem = BrowserMenuImageText(
-            context.getString(R.string.library_synced_tabs),
+        val syncSignIn = BrowserMenuImageText(
```

Comment:
Same nit as above: Let's leave this as part of the other PR. This is a half change as well because we need to update the `onItemTapped.invoke(ToolbarMenu.Item.SyncedTabs)` as well.

Reply:
I agree. The name change from synced tabs to sync sign in shouldn't be part of this card. Creating the invocation for ToolbarMenu.Item.SyncSignIn (or whatever it will be called) and changing that name will be part of https://github.com/mozilla-mobile/fenix/issues/17798